welly/well.py

```diff
@@ -161,11 +176,10 @@ def from_lasio(cls,
                    req=None,
                    alias=None,
                    fname=None,
-                   index=None,
-                   ):
+                   index=None):
```

Comment:
this should be formatted as is was, in line with PEP8.
I recommend, please install a proper linter.

Reply:
Done. linter is installed and this one wasn't reported.